reader_test.go

```diff
@@ -564,6 +566,35 @@ func TestMediaPlaylistWithOATCLSSCTE35Tag(t *testing.T) {
 	}
 }
 
+func TestMediaPlaylistWithEnvivioSCTE35Tag(t *testing.T) {
+	f, err := os.Open("sample-playlists/media-playlist-with-scte35-envivio.m3u8")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Should there be a `defer os.Close(f)` here?

Reply:
Good catch